lib/reline/kill_ring.rb

```diff
@@ -14,7 +14,7 @@ def initialize(str)
     end
 
     def ==(other)
-      object_id == other.object_id
+      equal?(other)
     end
```

Comment:
Alternatively, I could have deleted the `Struct#==` implementation with:

```suggestion
    remove_method(:==)
```

This would cause the  `Object#==` implementation to be inherited (which is identity-based), but it's much less clear, though.